index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="header">
+        <img class="logo" src="images/image_geometry_1.png" alt="logo">
+        <img class="mobile-menu" src="images/ic_menu.svg" alt="Open Navigation">
```

Comment:
- Please wrap your heading into a header as it would be better to use more semantic HTML tags.-